src/components/ui/team.tsx

```diff
@@ -0,0 +1,55 @@
+"use client";
+
+import React from "react";
+import Image from "next/image";
+import teamData from "@/data/team.json";
+
+const TeamSection = () => {
+  return (
+    <section className="wave-top wave-bottom">
+      {/* Content Section */}
+      <div
+        className="py-16 md:py-24 bg-coolGray-900"
+        style={{
+          backgroundImage: "url('/flex-ui-assets/elements/pattern-dark2.svg')",
+          backgroundPosition: "center",
+        }}
+      >
+        <div className="container px-4 mx-auto">
+          {/* Departments */}
+          {teamData.departments.map((department, departmentIndex) => (
+            <div key={departmentIndex} className="mb-12">
+              <h4 className="text-2xl font-bold text-white mb-8">{department.name}</h4>
+              <div className="flex flex-wrap -mx-4">
+                {department.members.map((member, memberIndex) => (
+                  <div
+                    key={memberIndex}
+                    className="w-full sm:w-1/2 lg:w-1/3 px-4 mb-12"
+                  >
+                    <div className="max-w-[250px] mx-auto font-default">
+                      <Image
+                        src={member.image}
+                        alt={member.name}
+                        width={150}
+                        height={150}
+                        className="block mb-6 rounded-full object-cover"
```

Comment:
These images don't need to be rounded and should be bigger. See:
![Screenshot 2024-12-10 at 7 11 01 AM](https://github.com/user-attachments/assets/26dca7e5-8cb3-44aa-ae1b-c2bbba09ca20)